frontend/src/components/app-config/user-config-form.tsx

```diff
@@ -1341,43 +1383,46 @@ export const UserConfigForm: React.FC = () => {
                 control={form.control}
                 disabled={isWasmRuntime}
                 name="ai.open_ai.model"
-                render={({ field }) => (
-                  <div className="flex flex-col space-y-1">
-                    <FormItem className={formItemClasses}>
-                      <FormLabel>Model</FormLabel>
-                      <FormControl>
-                        <Input
-                          list="ai-model-datalist"
-                          data-testid="ai-model-input"
-                          className="m-0 inline-flex"
-                          placeholder="gpt-4-turbo"
-                          {...field}
+                render={({ field }) => {
+                  const modelDatalistId = React.useId();
```

Comment:
there was probably a warning you were trying to fix. its ok to have that warning. we should remove `const modelDatalistId = React.useId()` instead since it might break react renders